client/client.go

```diff
@@ -269,6 +269,9 @@ type (
 	// QueryWorkflowWithOptionsResponse defines the response to QueryWorkflowWithOptions.
 	QueryWorkflowWithOptionsResponse = internal.QueryWorkflowWithOptionsResponse
 
+	// DescribeWorkflowExecution defines the response to DescribeWorkflow.
+	DescribeWorkflowExecution = internal.WorkflowExecutionDescription
```

Comment:
```suggestion
	WorkflowExecutionDescription = internal.WorkflowExecutionDescription
```

I think the internal name is a good one to export as

Reply:
Yeah not sure why I changed the name here